install script no longer works.

Looks like statsite has changed their install procedure.

```
.........
+ cd ../statsite
+ pip install --egg SCons
Downloading/unpacking SCons
  Downloading scons-2.4.1.tar.gz (639kB): 639kB downloaded
  Running setup.py (path:/tmp/pip_build_root/SCons/setup.py) egg_info for package SCons

Installing collected packages: SCons
  Running setup.py install for SCons
    changing mode of build/scripts/scons from 644 to 755
    changing mode of build/scripts/sconsign from 644 to 755
    changing mode of build/scripts/scons-time from 644 to 755
    changing mode of build/scripts/scons.bat from 644 to 755

    Installed SCons library modules into /usr/local/lib/scons-2.4.1
    Installed SCons scripts into /usr/local/bin
    Installed SCons man pages into /usr/local/man/man1
Successfully installed SCons
Cleaning up...
+ make
make: *** No targets specified and no makefile found.  Stop.

```

--- a/install
+++ b/install
@@ -26,7 +26,7 @@ fi
 apt-get update -y
 
 # Install package dependencies from apt
-RUNLEVEL=1 apt-get install -y libcairo2-dev libffi-dev pkg-config python-dev python-pip fontconfig apache2 libapache2-mod-wsgi git-core collectd memcached gcc g++ make
+RUNLEVEL=1 apt-get install -y libcairo2-dev libffi-dev pkg-config python-dev python-pip fontconfig apache2 libapache2-mod-wsgi git-core collectd memcached gcc g++ make libtool automake
 
 # Download source repositories for Graphite/Carbon/Whisper and Statsite
 cd /usr/local/src
@@ -39,7 +39,7 @@ git clone https://github.com/armon/statsite.git
 cd whisper; git checkout ${GRAPHITE_RELEASE}; python setup.py install
 cd ../carbon; git checkout ${GRAPHITE_RELEASE}; pip install -r requirements.txt; python setup.py install
 cd ../graphite-web; git checkout ${GRAPHITE_RELEASE}; pip install -r requirements.txt; python check-dependencies.py; python setup.py install
-cd ../statsite; pip install --egg SCons; make; cp statsite /usr/local/sbin/; cp sinks/graphite.py /usr/local/sbin/statsite-sink-graphite.py
+cd ../statsite; ./bootstrap.sh; ./confiure; make; cp src/statsite /usr/local/sbin/; cp sinks/graphite.py /usr/local/sbin/statsite-sink-graphite.py
 
 # Install configuration files for Graphite/Carbon and Apache
 cp ${SYNTHESIZE_HOME}/templates/statsite/statsite.conf /etc/statsite.conf

--- a/install
+++ b/install
@@ -39,7 +39,7 @@ git clone https://github.com/armon/statsite.git
 cd whisper; git checkout ${GRAPHITE_RELEASE}; python setup.py install
 cd ../carbon; git checkout ${GRAPHITE_RELEASE}; pip install -r requirements.txt; python setup.py install
 cd ../graphite-web; git checkout ${GRAPHITE_RELEASE}; pip install -r requirements.txt; python check-dependencies.py; python setup.py install
-cd ../statsite; ./bootstrap.sh; ./confiure; make; cp src/statsite /usr/local/sbin/; cp sinks/graphite.py /usr/local/sbin/statsite-sink-graphite.py
+cd ../statsite; ./bootstrap.sh; ./configure; make; cp src/statsite /usr/local/sbin/; cp sinks/graphite.py /usr/local/sbin/statsite-sink-graphite.py
 
 # Install configuration files for Graphite/Carbon and Apache
 cp ${SYNTHESIZE_HOME}/templates/statsite/statsite.conf /etc/statsite.conf

--- a/uninstall
+++ b/uninstall
@@ -27,7 +27,7 @@ rm /etc/init/statsite.conf
 rm /etc/cron.hourly/graphite-build-index
 
 # Uninstall our package dependencies and purge configurations
-apt-get purge -y python-cairo python-django python-django-tagging python-twisted python-zope.interface fontconfig apache2 libapache2-mod-wsgi python-pysqlite2 python-simplejson python-memcache git-core collectd memcached gcc g++ make
+apt-get purge -y python-cairo python-django python-django-tagging python-twisted python-zope.interface fontconfig apache2 libapache2-mod-wsgi python-pysqlite2 python-simplejson python-memcache git-core collectd memcached gcc g++ make libtool automake
 apt-get autoremove -y
 
 # Brute force cleanup on the collectd configuration directory

